donation-core/src/main/java/com/donation/VideoService.java

```diff
@@ -0,0 +1,18 @@
+package com.donation;
+
+import com.donation.donor.model.Video;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.List;
+
+public interface VideoService {
+
+    void emitVideoFrames(final InputStream is, OutputStream os) throws IOException;
+
+    List<Video> getVideosByProject(Long idProject);
```

Comment:
Do we really need this method: getVideosByProject
I think it could be a private method in VideoServiceImpl

Reply:
Done